docs/middle_low_german.rst

```diff
@@ -0,0 +1,33 @@
+Middle Low German
+*****************
+Middle Low German or Middle Saxon is a language that is the descendant of Old Saxon and the ancestor of modern Low German. It served as the international lingua franca of the Hanseatic League. It was spoken from about 1100 to 1600, or 1200 to 1650.
+(Source: `Wikipedia <https://en.wikipedia.org/wiki/Middle_Low_German>`_)
+
+POS Tagging
+===========
+
+The POS taggers were trained by NLTK's models.
+
```

Comment:
Please make a note here of where the data came from.

Reply:
Otherwise this addition is looking great, thanks for making this addition.

Reply:
Do I need to provide full citations (like in the [readme](https://github.com/cltk/middle_low_german_models_cltk/blob/master/README.md)) or simply link the source?

Reply:
just a simple link should suffice